ecpp

```diff
@@ -112,7 +138,95 @@ def main():
     except KeyboardInterrupt:
         print(n)
         raise
-    print("Number is prime")
+    print("\n")
+    print("[+] number is prime\n")
+
+def correlate(moduli, certs=None, verify_opt=None, primo_opt=None):    
+    missing_cert_flag = False 
+    cert_files = []
+    if certs is None:
+        certs = certs_dir
+    for path, subdirs, files in os.walk(certs):
+        for name in files:
+            cert_files.append(os.path.join(path, name))
+    cert_for_moduli  = []
+    no_cert_for_moduli = []
+    status = None
+    cert_dict = {}
+    for cert in cert_files:
+        if cert.endswith('.out'):
+            path = cert
+            cert_candidate = candidate_finder(path)
+            cert_dict[cert_candidate] = path
+    with open(str(moduli)) as f:
+        inputfile = filter(None, (line.rstrip() for line in f))
+        for count_line, line in enumerate(inputfile, 1):
+            if not line.startswith('#'):
+                prime_moduli = (line.split(" ")[6])
+                int_prime_moduli = int(prime_moduli, 16)
+                status = False
+                print("=====================================================")
+                print("Searching cert for: {0}".format(int_prime_moduli))
+                if int_prime_moduli in cert_dict:
+                            path = cert_dict[int_prime_moduli]
+                            print("[+] line {0}: {1}".format(count_line, path))
+                            status = True
+                            if verify_opt == True:
+                                verify(path)
+                if status == False:
+                    print("[-] line {0}: no certificate found".format(count_line))
+                    missing_cert_flag = True
+                    if primo_opt:
+                        print("[+] line {0}: creating input file for {1}".format(count_line,primo_in))
```

Comment:
space missing in format after comma, also the line is too long, it should not exceed 80 chars